great_expectations/dataset/pandas_dataset.py

```diff
@@ -1313,5 +1387,70 @@ def expect_column_kl_divergence_to_be_less_than(self, column, partition_object=N
                     }
                 }
 
-
         return return_obj
+
+
+    @DocInherit
+    @MetaPandasDataset.column_pair_map_expectation
+    def expect_column_pair_values_to_be_equal(self,
+        column_A,
+        column_B,
+        ignore_row_if="both_values_are_missing",
+        output_format=None, include_config=False, catch_exceptions=None
+    ):
+        return column_A == column_B
+
+    @DocInherit
+    @MetaPandasDataset.column_pair_map_expectation
+    def expect_column_pair_values_A_to_be_greater_than_B(self,
+        column_A,
+        column_B,
+        or_equal=None,
+        parse_strings_as_datetimes=None,
+        allow_cross_type_comparisons=None,
+        ignore_row_if="both_values_are_missing",
+        output_format=None, include_config=False, catch_exceptions=None
+    ):
+        #FIXME: Implement allow_cross_type_comparisons
+
+        if parse_strings_as_datetimes:
+            temp_column_A = column_A.map(parse)
+            temp_column_B = column_B.map(parse)
+
+        else:
+            temp_column_A = column_A
+            temp_column_B = column_B
+
+
+        if or_equal:
```

Comment:
or_equal == True, to be explicit since default is None? Test for nonsense values there and warn?

Reply:
Good catch.